### ISSUE TITLE
WCAG issue on input border and placeholder

### ISSUE THREAD

**Issue by jonathanulco** [tailwindlabs/tailwindcss-forms#3]
Hi Tailwind's Team 

According to wcag 2.1 graphic elements should have a contrast of at least 3:1 
A border colour with a higher contrast than #d1d5db should be proposed as a default.

The same goes for the contrasts of the placeholders. These should have a contrast of at least 4.5:1.

https://www.w3.org/TR/WCAG21/#non-text-contrast 

https://www.w3.org/TR/WCAG21/#contrast-minimum

**Comment by adamwathan** [comment]
Updated the styles to be much more "raw" and encourage additional styling, and made sure to update these colors to meet those guidelines in the process. If we're going to encourage people to do the styling themselves then really no reason not to 👍

https://tailwindcss-forms.vercel.app/

https://github.com/tailwindlabs/tailwindcss-forms/commit/3288709b59f4101511ec19f30cb2dafe7738251e

**Comment by mattgreenfield** [comment]
I've noticed that the placeholder color now differs from the tailwind default (set in preflight).
- Do you think this fix should also be applied to preflight?
- Should this typography plugin even be setting a placeholder color if its already in preflight?
  - Possibly not as some users don't use preflight, but those that do are getting more CSS than they need 🤷
